.env.example

```diff
@@ -3,3 +3,5 @@ REACT_APP_AUTHENTICATED=[yes|no]
 REACT_APP_API_URL=http://localhost:3001/
 REACT_APP_OPENID_CLIENT_ID=1234
 REACT_APP_OPENID_AUTHORITY=https://api.hel.fi/sso-test/openid/
+REACT_APP_OPEN_ID_API_TOKENS_URL=https://api.hel.fi/sso-test/api-tokens/
```

Comment:
Should it just be `REACT_APP_SSO_URL` and just use `/openid/` or `/api-tokens/`with in code 🤔